example/tests/elementActions.test.js

```diff
@@ -1,12 +1,20 @@
 import { Element } from "test-juggler";
+const fs = require("fs").promises;
+const fsExtra = require("fs-extra");
 
 describe("Element Actions", () => {
     const sliceToClick = new Element("[seriesName='seriesx2'] path");
+    const localPath = process.cwd().replace(/\\/g, "/");
 
     beforeEach(async () => {
         console.log("Running test: " + jasmine["currentTest"].fullName);
     });
 
+    afterAll(async () => {
+        const tempFileDir = process.cwd() + "\\example\\testFiles\\temp";
```

Comment:
I think \\ will only work on Windows. Can we use / here instead? same in lines 313 328

Reply:
Yes, we can. Fixed.